x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/actions_popover.test.tsx

```diff
@@ -119,14 +119,33 @@ describe('ActionsPopover', () => {
       />
     );
 
-    expect(getByRole('link')?.getAttribute('href')).toBe('/a/test/edit/url');
+    expect(getByTestId('editMonitorLink')?.getAttribute('href')).toBe('/a/test/edit/url');
+  });
+
+  it('contains link to clone monitor', async () => {
+    jest
+      .spyOn(editMonitorLocatorModule, 'useEditMonitorLocator')
+      .mockReturnValue('/a/test/edit/url');
+    const { getByTestId } = render(
+      <ActionsPopover
+        position="relative"
+        isPopoverOpen={true}
+        setIsPopoverOpen={jest.fn()}
+        monitor={testMonitor}
+        locationId={testMonitor.location.id}
+      />
+    );
+
+    expect(getByTestId('cloneMonitorLink')?.getAttribute('href')).toBe(
+      'synthetics/add-monitor?cloneId=1lkjelre'
+    );
   });
 
   it('contains link to detail page', async () => {
     jest
       .spyOn(monitorDetailLocatorModule, 'useMonitorDetailLocator')
       .mockReturnValue('/a/test/detail/url');
-    const { getByRole } = render(
+    const { getByTestId } = render(
```

Comment:
Like the other instances here, can we can pick up the edit link via its text?

This is not that important but good practice to rely on accessible attributes where possible vs. using the test id.